src/code42cli/cmds/search/cursor_store.py

```diff
@@ -75,13 +76,31 @@ def __init__(self, profile_name):
         super().__init__(dir_path)
 
 
-def get_file_event_cursor_store(profile_name):
-    return FileEventCursorStore(profile_name)
+class AuditLogCursorStore(BaseCursorStore):
```

Comment:
There should be test class `tests.cmds.search.test_cursor_store.TestAuditLogCursorStore` but I am not seeing it

Reply:
Added these.